src/styles/blocks/header.scss

```diff
@@ -0,0 +1,70 @@
+@import 'variables';
+
+body {
+  margin: 0;
+  font-size: 12px;
+  font-weight: 500;
+  font-family: Roboto, sans-serif;
+}
+
+*,
+*::after {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
```

Comment:
- Move to normilize/reset file as it's not related to header
- Don't set margins/paddins to all elements, you can reset only for needed
![image](https://github.com/mate-academy/layout_catalog/assets/63243136/5db6ffa4-cf2d-434e-ba64-397fd1c11e9c)